go.mod

```diff
@@ -7,29 +7,28 @@ require (
 	github.com/Microsoft/go-winio v0.6.1
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/blang/semver/v4 v4.0.0
-	github.com/checkpoint-restore/checkpointctl v0.1.0
-	github.com/checkpoint-restore/go-criu/v6 v6.3.0
-	github.com/container-orchestrated-devices/container-device-interface v0.6.0
```

Comment:
This change follows the following upstream update:

- https://github.com/cncf-tags/container-device-interface/pull/172

And, when included, would resolve package import issues.